samples/bluetooth/iso_broadcast_benchmark/src/broadcaster.c

```diff
@@ -24,13 +24,13 @@ LOG_MODULE_REGISTER(iso_broadcast_broadcaster, LOG_LEVEL_DBG);
 #define DEFAULT_BIS_COUNT       CONFIG_BT_ISO_MAX_CHAN
 
 NET_BUF_POOL_FIXED_DEFINE(bis_tx_pool, CONFIG_BT_ISO_TX_BUF_COUNT,
-			  DEFAULT_BIS_SDU, NULL);
+			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU), NULL);
 
 static K_SEM_DEFINE(sem_big_complete, 0, 1);
 static K_SEM_DEFINE(sem_big_term, 0, 1);
 static struct k_work_delayable iso_send_work;
 static uint32_t iso_send_count;
-static uint8_t iso_data[CONFIG_BT_ISO_TX_MTU - BT_ISO_CHAN_SEND_RESERVE];
+static uint8_t iso_data[DEFAULT_BIS_SDU];
```

Comment:
This one confused me - DEFAULT_BIS_SDU is used for the data array, but CONFIG_BT_ISO_TX_MTU for the buffer that will (?) be used for the data.
Please enlighten me.

Reply:
`DEFAULT_BIS_SDU` is equal to `CONFIG_BT_ISO_TX_MTU`, so it doesn't really matter which one we use here. Please let me know if you want me to change it back to `CONFIG_BT_ISO_TX_MTU` - I don't have any strong preferences. 

The buffer actually uses `BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU)` which is larger than `CONFIG_BT_ISO_TX_MTU)`.

Reply:
(Yes, I know, the header is also added.  I was talking about the data part.)
As they are the same, the sizes are fine.

It also makes sense to use the BIS one in a broadcaster.  So I guess the connection obvious it might be clearest to use `DEFAULT_BIS_SDU` also as the argument to  `BT_ISO_SDU_BUF_SIZE()`,

Reply:
Mixing what you are using as a define here. Not a problem at the moment since they are the same, but messy and could lead to maintenance issues. Suggest to remove DEFAULT_BIS_SDU or always use it.

Reply:
Changed the array declaration to use `CONFIG_BT_ISO_TX_MTU`. I'm keeping `DEFAULT_BIS_SDU` as that is used for parameter configuration information.